### ISSUE TITLE
Symbol prop keys are not working

### ISSUE THREAD

**Issue by adipascu** [JS-DevTools/ono#8]
Thank you for the great library!
I have a usecase with symbols that is not currently working, here is some sample code:
```js
const ono = require('ono');

const SOME_PROP = Symbol('symbol-name');
// const SOME_PROP = 'string-key'; // (string keys work correctly)
function fun() {
  try {
    throw new Error('original error');
  } catch (e) {
    throw ono(e, { [SOME_PROP]: true }, 'wrapped with [SOME_PROP]');
  }
}

try {
  fun();
} catch (e) {
  console.log(`Has prop: ${e[SOME_PROP] || false}`);
}

```

**Comment by JamesMessinger** [comment]
Good catch!  I never even thought about supporting symbol props, but that totally makes sense.

**Comment by JamesMessinger** [comment]
We just released Ono v5.0, which fixes this issue.  Thanks for reporting it!

[Here are the release notes](https://github.com/JS-DevTools/ono/blob/master/CHANGELOG.md)
